games-arcade/caveexpress/patches/caveexpress-2.4.patchset

```diff
@@ -0,0 +1,22 @@
+From 900ef70f6134aa2d52e10c78d5242aa03c0cafca Mon Sep 17 00:00:00 2001
+From: Ra-V2 <rav.bernacki@gmail.com>
+Date: Tue, 4 Feb 2020 00:32:30 +0100
+Subject: fix build
+
+
+diff --git a/src/modules/common/LUA.h b/src/modules/common/LUA.h
+index 5de5b79..0a4c5b2 100644
+--- a/src/modules/common/LUA.h
++++ b/src/modules/common/LUA.h
+@@ -3,7 +3,7 @@
+ #include "common/Config.h"
+ GCC_DIAG_OFF(cast-qual)
+ GCC_DIAG_OFF(cast-align)
+-#include <lua.hpp>
++#include <lua52/lua.hpp>
```

Comment:
This is unneeded if you craft your includes correctly.
You probably want something like this in your recipe:
```
export CFLAGS="`pkg-config --cflags lua`"
```